miniapp/USERGUIDE.md

```diff
@@ -612,6 +612,80 @@ dependency {
 ```
 </details>
 
+<details><summary markdown="span"><b>How can I use this SDK in a Java project?</b>
+</summary>
+
+We don't support usage of the Mini App SDK in a Java project. This is because this SDK uses Kotlin specific features such as `suspend` functions and `Coroutines`.
+
+However, it is possible to use this SDK in a Java project if you wrap the calls to `suspend` functions into something that Java can use such as a `Future` or a normal callback interface.
+
+To do this, you will need to integrate Kotlin and Kotlin Coroutines into your project:
+
+```groovy
+// In your top level "build.gradle" file:
+buildscript {
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.50"
+    }
+}
+
+// In your project "myApp/build.gradle" file
+apply plugin: 'kotlin-android'
+
+dependencies {
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.3.50"
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-jdk8:1.3.0"
+}
+```
+
+Then, you will need to create wrapper functions for the Mini App SDK functionality which you wish to use.
+
+If your minimum SDK level is 24 or higher, then you can use Java `CompletableFuture`:
```

Comment:
Also, I think we can link https://developer.android.com/reference/java/util/concurrent/CompletableFuture here.